.github/workflows/validateindex.yml

```diff
@@ -6,9 +6,35 @@ jobs:
   verify-json-validation:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Validate JSON
-        uses: docker://orrosenblatt/validate-json-action:latest
-        env:
-          INPUT_SCHEMA: ./index.schema
-          INPUT_JSONS: ./INDEX
+    - uses: actions/checkout@v1
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install jsonschema
+
+    - name: Validate INDEX JSON
+      run: jsonschema -i ./INDEX ./index.schema
+
+    - name: Validate plugins JSONs
+      run: |
+        wget https://raw.githubusercontent.com/fulder/iocage/plugin-schema-fixes/iocage_lib/plugin_manifest.json -O plugin_manifest.schema
```

Comment:
Let's please change this to upstream iocage again

Reply:
Thanks for the really fast merge in `iocage` repository :+1:. Changed back to upstream master again.